class.photon.php

```diff
@@ -579,17 +579,24 @@ public function filter_image_downsize( $image, $attachment_id, $size ) {
 	 * @uses self::validate_image_url, jetpack_photon_url
 	 * @return array An array of Photon image urls and widths.
 	 */
-	public function filter_srcset_array( $sources ) {
+	public function filter_srcset_array( $sources, $size_array, $image_src, $image_meta, $attachment_id ) {
```

Comment:
They can be removed from both lines. Added to make exploring options to fix faster, but in the end nothing helpful there for the final proposal. Thanks for catching it, can remove in a few hours if you don't get to it first.